Tchap/Managers/ThirdPartyIDPlatformInfoResolver/IdentityServersURLGetter.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ Copyright 2018 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+
+/// `IdentityServersURLGetter` is used to retrieved all the known identity server urls.
+final public class IdentityServersURLGetter {
+    
+    // MARK: - Public
+    
+    // The available identity servers
+    public var identityServerUrls = [String]()
```

Comment:
Use `public let identityServerUrls: [String] = []`. To use `let` here define a local `identityServerUrls` variable in init and then make `self.identityServerUrls = identityServerUrls`. I find `[String] = []` more readable than `[String]()` although it do the same thing.